src/cluster_legacy.c

```diff
@@ -2943,6 +2952,13 @@ int clusterProcessPacket(clusterLink *link) {
     if (type == CLUSTERMSG_TYPE_PING || type == CLUSTERMSG_TYPE_PONG || type == CLUSTERMSG_TYPE_MEET) {
         serverLog(LL_DEBUG, "%s packet received: %.40s", clusterGetMessageTypeString(type),
                   link->node ? link->node->name : "NULL");
+
+        if (sender && (sender->flags & CLUSTER_NODE_MEET)) {
+            /* Once we get a response for MEET from the sender, we can stop sending more MEET. */
+            sender->flags &= ~CLUSTER_NODE_MEET;
+            serverLog(LL_NOTICE, "Successfully completed handshake with %.40s (%s)", sender->name,
+                      sender->human_nodename);
```

Comment:
Not a blocker for this PR but I think it will be good if we could standardize the logging statement, at least in cluster.c and cluster_legacy.c (which by the way should be renamed as we don't intend to deprecate the current cluster implementation any time soon, even considering the V2 work). There has been discussion in the past about using a more "structural" logging mechanism as well but I now think "incremental perfection" is more practical. What I would like to see in the server log and int the cluster topology area is:

"who" received a message from "whom" about "which node"'s "what" properties have changed from "which value" to "which other value" at "when".

Ideally, an admin can just look at a single log statement to gather all the information.

Reply:
moved the discussion to #656